src/_guides/language/language-tour.md

```diff
@@ -2206,10 +2212,9 @@ your exception handler needs the exception object.
 
 You can specify one or two parameters to `catch()`.
 The first is the exception that was thrown,
-and the second is the stack trace
-(a [StackTrace]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/StackTrace-class.html) object).
+and the second is the stack trace (a [StackTrace][] object).
 
-<?code-excerpt "language-tour/flow/exceptions/web/main.dart (try-catch-2)" replace="/\(e.*?\)/[!$&!]/g"?>
+<?code-excerpt "lib/language_tour/exceptions.dart (try-catch-2)" replace="/\(e.*?\)/[!$&!]/g"?>
```

Comment:
Please add something about `[!` to the README. (It could point elsewhere... I just want to make sure that every weird bit of syntax we use in these examples is easily findable in the README.)

Reply:
done